main_logic/core.py

```
@@ -2032,28 +2215,43 @@ async def feed_tts_chunk(self, text: str):
+        try:
+            async with self._proactive_write_lock:
+                delivered = await self.send_lanlan_response(full_text, is_first_chunk=True)
 
-            from utils.llm_client import AIMessage as _AIMsg
-            if self.session and hasattr(self.session, '_conversation_history'):
-                self.session._conversation_history.append(_AIMsg(content=full_text))
+                if delivered:
+                    if self.use_tts and self.tts_thread and self.tts_thread.is_alive():
+                        try:
+                            self.tts_request_queue.put((None, None))
+                        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)